hibernate-core/src/test/java/org/hibernate/test/locking/AbstractSkipLockedTest.java

```diff
@@ -71,6 +70,7 @@ public void testSQLServerSkipLocked() {
 
 	@Test
 	@RequiresDialect({ PostgreSQL95Dialect.class })
+	@RequiresDialectFeature(DialectChecks.SupportSkipLocked.class)
```

Comment:
I think `@RequiresDialectFeature` can be removed

Reply:
done